### PR TITLE
bonsai integration for pagerduty plugin

### PR DESCRIPTION
**Pull request by sreejita-biswas**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [ ] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,17 +16,23 @@ script:
 - bundle exec rake default
 - gem build sensu-plugins-pagerduty.gemspec
 - gem install sensu-plugins-pagerduty-*.gem
+ibefore_deploy:
```
Comment (majormoses) [inline .travis.yml]: the `i` looks like an accident.

File: sensu-plugins-pagerduty.gemspec [hunk 2]
```diff
@@ -23,12 +23,12 @@ Gem::Specification.new do |s|
   s.platform               = Gem::Platform::RUBY
   s.post_install_message   = 'You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu'
   s.require_paths          = ['lib']
-  s.required_ruby_version  = '>= 2.0.0'
+  s.required_ruby_version  = '>= 2.3.0'
```
Comment (majormoses) [inline sensu-plugins-pagerduty.gemspec:26]: We need to call this out as a breaking change see https://github.com/sensu-plugins/sensu-plugins-redis/pull/69/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR10 for an example.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+- Bump `sensu-plugin` dependency from `~> 2.5` to `~> 4.0` you can read the changelog entries for [4.0](https://github.com/sensu-plugins/sensu-plugin/blob/master/CHANGELOG.md#400---2018-02-17), [3.0](https://github.com/sensu-plugins/sensu-plugin/blob/master/CHANGELOG.md#300---2018-12-04), and [2.0](https://github.com/sensu-plugins/sensu-plugin/blob/master/CHANGELOG.md#v200---2017-03-29)
```
Comment (majormoses) [inline CHANGELOG.md:8]: This needs to go under a `### Breaking Changes` header. see https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#the-breaking-change

**Review by majormoses — CHANGES_REQUESTED** [review]
This is awesome thanks for this. I left some suggestions. I suggest @jspaleta looks over the bonsai specific config and get his 👍

File: .travis.yml (hunk 1 above)
Comment (sreejita-biswas) [reply .travis.yml]: I have updated the same.

File: sensu-plugins-pagerduty.gemspec (hunk 2 above)
Comment (sreejita-biswas) [reply sensu-plugins-pagerduty.gemspec:26]: updated the same

File: CHANGELOG.md (hunk 3 above)
Comment (sreejita-biswas) [reply CHANGELOG.md:8]: updated the same.

**Comment by msmith-hpe** [comment]
bump...

**Comment by majormoses** [comment]
Hey sorry for the delay I was out all last week at conferences (including sensu summit). Let me take a look.

**Review by majormoses — APPROVED** [review]
LGTM

**Comment by majormoses** [comment]
Release has been cut awaiting CI + CD

**Comment by majormoses** [comment]
@sreejita-biswas Thanks for your contribution to Sensu plugins! Without people like you submitting PRs we couldn't run the project.

**Comment by majormoses** [comment]
released:
- https://rubygems.org/gems/sensu-plugins-pagerduty/versions/5.0.0

@jspaleta can you look into this issue with the bonsai release? https://travis-ci.org/sensu-plugins/sensu-plugins-pagerduty/jobs/585855470#L1268

**Pull request merged**